main.js

```diff
@@ -12,9 +12,42 @@ window.addEventListener('load', trigger.bind(null, 'load'));
 window.addEventListener('load', trigger.bind(null, 'DOMContentLoaded'));
 document.addEventListener('DOMContentLoaded', trigger.bind(null, 'DOMContentLoaded'));
 
+document.onreadystatechange = function () {
+	if (document.readyState === 'complete') {
+		trigger('DOMContentLoaded');
+		trigger('load');
+	}
+};
+
 if (document.readyState === 'complete') {
-	trigger('load');
-	trigger('DOMContentLoaded');
-} else if (document.readyState === 'interactive') {
```

Comment:
Can you describe the problem that this causes?  I'm concerned that the solution below will delay o.domcontentloaded in browsers that support the `interactive` readyState.